### ISSUE TITLE
[Feature Request] Multiple Inputs

### ISSUE THREAD

**Issue by L3V147H4N** [aakashns/react-native-dialogs#35]
I think there is only one thing missing from this, either in the implementation or in the docs, a Dialog with Multiple inputs.

Many TextInputs or Text + Checkbox, etc

![image](https://cloud.githubusercontent.com/assets/5846479/17442225/21a79030-5b0b-11e6-8371-c974bfc6cbd3.png)

**Comment by BigPun86** [comment]
+1

**Comment by Mayzie** [comment]
Yeah, was looking for ages on how to do support multiple inputs with this RN project, expecting this "wrapper" allowing it, considering the underlying library does.

Oh wells.

**Comment by Noitidart** [comment]
I was also trying multiple, but not necessarily multiple inputs, but multi fields. I wanted to show content, one input, and at bottom a checkbox for "dont ask me again".
